Makefile

```diff
@@ -332,6 +332,15 @@ generate-manifests-core: $(CONTROLLER_GEN)
 		crd:crdVersions=v1 \
 		output:crd:dir=./cmd/clusterctl/config/crd/bases
 
+.PHONY: generate-manifests-operator
+generate-manifests-operator: $(CONTROLLER_GEN)
```

Comment:
We have a separate makefile for the operator in `exp/operator`. One of the reasons is to avoid changes outside of the operator directory while it's being developed in a separate branch.